.travis.yml

```diff
@@ -0,0 +1,12 @@
+before_install:
+  - nvm install 8
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.0.2
+  - export PATH=$HOME/.yarn/bin:$PATH
+
+cache:
+  yarn: true
+
+install:
+  - yarn
```

Comment:
could we use npm rather than yarn? as a rule of thumb, I'd rather not force introducing yarn since we're mostly just using npm all over the place

Reply:
As for this, I've copied these from metal-clay-components' `.travis.yml`, but will amend it.

Reply:
Right, we're using it there to workaround the mono-repo restrictions